src/api/fixtures/buildPayTransactionFixtures.ts

```diff
@@ -0,0 +1,38 @@
+export const MOCK_VALID_CHARGE_ID = '1b03e80d-4e87-46fd-9772-422a1b693fb7';
```

Comment:
Oh yeah, let's have the mocks in the `src/api/mocks.ts` file.

Like we have in https://github.com/coinbase/onchainkit/blob/main/src/swap/mocks.ts